teams/teams-ps/teams/Update-CsCustomPolicyPackage.md

```diff
@@ -0,0 +1,98 @@
+---
+external help file: Microsoft.TeamsCmdlets.PowerShell.Custom.dll-Help.xml
+Module Name: MicrosoftTeams
+title: Update-CsCustomPolicyPackage
+author: gucsun
+ms.author: gucsun
+manager: amitar
+online version: https://docs.microsoft.com/powershell/module/teams/update-CsCustomPolicyPackage
+schema: 2.0.0
+---
+
+# Update-CsCustomPolicyPackage
+
+## SYNOPSIS
+This cmdlet submits an operation that updates a custom policy package with new package settings.
+
+## SYNTAX
+
+```
+Update-CsBatchPolicyPackageAssignmentOperation -Identity <String> -PolicyList <String[]> [-Description <String>] [<CommonParameters>]
+```
+
+## DESCRIPTION
+
+This cmdlet submits an operation that updates the policies into a custom policy package. It allows the user to change the description and policies included in the package. The available policy types can be found here https://docs.microsoft.com/MicrosoftTeams/manage-policy-packages.
```

Comment:
can rephrase: This cmdlet submits an operation that updates a custom policy package with new package settings.